usaspending_api/awards/management/commands/helpers.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+from django.db.models import Q
+
+from usaspending_api.awards.models import Award
+from usaspending_api.references.models import Agency, Location, RefCountryCode
+
+
+def up2colon(input_string):
+    'Takes the part of a string before `:`, if any.'
+
+    return input_string.split(':')[0].strip()
+
+
+def get_agency(self, agency_string):
+    agency_code = up2colon(agency_string)
+    agency = Agency.objects.filter(
+        subtier_agency__subtier_code=agency_code).first()
+    if not agency:
+        self.logger.error("Missing agency: " + agency_string)
+    return agency
+
+
+def convert_date(date):
+    return datetime.strptime(date, '%m/%d/%Y').strftime('%Y-%m-%d')
+
+
+def fetch_country_code(vendor_country_code):
+    code_str = up2colon(vendor_country_code)
+
+    country_code = RefCountryCode.objects.filter(
+        Q(country_code=code_str) | Q(country_name__iexact=code_str)).first()
+    if not country_code:
+        # We don't have an exact match on the name or the code, so we need to
+        # chain filter on the name
+        query_set = RefCountryCode.objects
+        for word in code_str.split():
+            query_set = query_set.filter(country_name__icontains=word)
+        country_code = query_set.first()
+
+    return country_code
+
+
+def get_or_create_award(row):
+    piid = row.get("piid", None)
+    parent_award_id = row.get("idvpiid", None)
```

Comment:
That's right, I'd forgotten about that.  I'll move this back to loadcontracts after all.